toonz/sources/image/ffmpeg/tiio_gif.cpp

```diff
@@ -92,7 +92,7 @@ TLevelWriterGif::~TLevelWriterGif() {
   preIArgs << "-v";
   preIArgs << "warning";
   preIArgs << "-r";
-  preIArgs << QString::number(m_frameRate);
+  preIArgs << QString::number(std::max(m_frameRate, 12.0));
```

Comment:
Can a gif have a lower fps than 12?

Reply:
Would it be better at:
preIArgs << QString::number((m_frameRate < 1) ? 12 : m_frameRate);

Reply:
I have a gif that I downloaded from some site some time ago that has a r_frame_rate=100/10 (10fps) with a duration = 0.1 which calculates to a 1 frame gif, which it is, so it can be less than 12.

I prefer to preserve whatever frame rate is obtained from the file. If for some strange reason there isn't one, i'm assuming 12

Actually the only reason it was 0 was because the getFrameRate() was called before getFrameCount() and there was a condition to only get frame rate if frame count > 0 so it was never getting rate properly.

Reply:
Line 95 as it stands will overwrite a fps lower than 12 with 12.

Reply:
Think i meant min, but realize that will not help if it's 0. Will fix as suggested. Thanks for reviewing.